corehq/apps/locations/models.py

```diff
@@ -353,7 +353,7 @@ class SQLLocation(MPTTModel):
     metadata = jsonfield.JSONField(default=dict, blank=True)
     created_at = models.DateTimeField(auto_now_add=True)
     last_modified = models.DateTimeField(auto_now=True)
-    is_archived = models.BooleanField(default=False)
+    is_archived = models.BooleanField(default=False, db_index=True)
```

Comment:
wouldn't this only help if is_archived is the only column we're querying (which i don't think we do often)?

maybe is_archived should be added in a multicolumn index with the other columns (like maybe domain) depending on our use of it